.env.sample

```
@@ -0,0 +1,14 @@
+DATABASE_URL="mysql://username:password@127.0.0.1:3306/db_name"
```

dbmate를 적용하기 위해 DATABASE_URL을 잘 설정해주셨습니다.

다만, 데이터베이스의 호스트가 반드시 `127.0.0.1` 즉, localhost가 아닐 수도 있습니다.

또한 3306은 MySQL 데이터베이스의 고유 포트 번호이기 때문에 직접 명시하는 것 보다 아래와 같이 수정하여 주는 것이 좋을 것 같습니다.

배포 후에는 하나의 rdbms 클라이언트를 사용하지만 local 상에서 상이하게 적용될 수 있도 있기 때문입니다.

 ```suggestion
DATABASE_URL="mysql://username:password@host_number:port_number/db_name"
```